packages/perseus/src/util/tex.ts

```diff
@@ -16,27 +14,6 @@ function findChildOrAdd(elem: any, className: string) {
     return $child;
 }
 
-function doCallback(
-    elem: HTMLElement,
-    callback: (() => unknown) | (() => void),
-) {
-    let tries = 0;
-    (function check() {
-        const height = elem.scrollHeight;
-        // Heuristic to guess if the font has kicked in
-        // so we have box metrics (magic number ick,
-        // but this seems to work mostly-consistently)
-        if (height > 18 || tries >= 10) {
-            callback();
-        } else {
-            tries++;
-            // TODO(jeff, CP-3128): Use Wonder Blocks Timing API
-            // eslint-disable-next-line no-restricted-syntax
-            setTimeout(check, 100);
-        }
-    })();
```

Comment:
This hack was pretty horrible! We polled every 100ms waiting for the math fonts to load in so we could update the layout. We don't need to do this anymore, because MathJax 3 includes the font metrics in its JavaScript, so it can lay out the math accurately even before the fonts have loaded.

Reply:
Yeah. I'd spent some time working on this function a few years ago for mobile. Glad we can get rid of it today!!